### DISCUSSION TITLE
How to disable the Update parameter windows at starting

### DISCUSSION THREAD

**Discussion by StillTemporary** [mRemoteNG/mRemoteNG#2444] [Q&A] [answered]
Hello,

I have the last version (1.76.20.24669) on Windows 10.
Each time I launch mRemoteNG, the Update parameter window appears. Is there a way to disable it ?

Thanks !
![mremote_spam](https://github.com/mRemoteNG/mRemoteNG/assets/123190081/f81219b6-b4e4-4992-8715-c0d6fb42c807)

**Comment by Kvarkas** [comment, marked as answer]
Hello, click second bar and then set to dont check:
![image](https://github.com/mRemoteNG/mRemoteNG/assets/3611964/31ba05c3-f37e-439c-9647-ca8c795b763e)

**Reply by StillTemporary** [reply]
Thank you !!
